test/Makefile

```diff
@@ -15,7 +17,9 @@ endif
 
 test: clean test_base64 benchmark
 	./test_base64
+ifeq (, $(findstring mingw, $(TARGET)))  # no "/dev/urandom" on Windows
```

Comment:
There's a really old pull request, #6, which fixes this on Windows. Now that we have a provisional CI environment for Windows, I can try to revive that branch and see if I can get the benchmark to work.

Having a working Windows benchmark would also be great for checking in CI that certain changes did not introduce speed regressions on that platform.